src/Voron/Slice.cs

```diff
@@ -76,12 +76,6 @@ public Slice(byte[] key, ushort size)
             Array = key;
         }
 
-        public Slice(ulong* ptr):
```

Comment:
There isn't any other usage of it. Anyway, I prefer to specify the pointer and the size explicitly. Passing just a pointer is a bit misleading I think.